contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -157,9 +157,11 @@ private const val WRAP = ViewGroup.LayoutParams.WRAP_CONTENT
  */
 open class ContourLayout(
```

Comment:
Does `@JvmOverloads` also accomplish this or will it still only suggest the two arg version?

Reply:
It surprisingly doesn't and only suggests the two arg version

Reply:
Maybe add a quick comment to the secondary constructor about that? I would be so tempted to inline that again.

Reply:
Does this read good enough?

```
// Using a secondary constructor instead of @JvmOverloads ensures that
// it shows up in autocomplete suggestions when extending ContourLayout.
constructor(context: Context): this(context, null)
```

Reply:
SGTM